src/Gateways/GpApiGateway.php

```diff
@@ -136,6 +219,50 @@ protected function add_hooks() {
 		add_action( 'woocommerce_api_globalpayments_threedsecure_challengenotification', array( $this, 'process_threeDSecure_challengeNotification' ) );
 	}
 
+	public function gateway_title( $gateway_title, $gateway_id ) {
```

Comment:
TBH, this seems like a good enhancement that would help all gateways. We should move this up into `AbstractGateway`.